rest/config_database.go

```diff
@@ -90,17 +90,22 @@ func GenerateDatabaseConfigVersionID(ctx context.Context, previousRevID string,
 }
 
 func DefaultPerDBLogging(bootstrapLoggingCnf base.LoggingConfig) *DbLoggingConfig {
+	dblc := &DbLoggingConfig{}
 	if bootstrapLoggingCnf.Console != nil {
 		if *bootstrapLoggingCnf.Console.Enabled {
-			return &DbLoggingConfig{
-				Console: &DbConsoleLoggingConfig{
-					LogLevel: bootstrapLoggingCnf.Console.LogLevel,
-					LogKeys:  bootstrapLoggingCnf.Console.LogKeys,
-				},
+			dblc.Console = &DbConsoleLoggingConfig{
+				LogLevel: bootstrapLoggingCnf.Console.LogLevel,
+				LogKeys:  bootstrapLoggingCnf.Console.LogKeys,
 			}
 		}
 	}
-	return &DbLoggingConfig{}
+	if bootstrapLoggingCnf.Audit != nil {
+		dblc.Audit = &DbAuditLoggingConfig{
+			Enabled:       base.BoolPtr(false),
+			EnabledEvents: base.DefaultAuditEventIDs,
+		}
```

Comment:
I would have expected these to inherit the value from the bootstrap and override if appropriate?

Is this change specific to capella, since audit logging should be enabled globally, but potentially turned off per database?

Reply:
It's anticipated that users will enable audit logging (globally), and then opt-into databases one at a time where required. Enabling globally then disabling subsets of databases creates unnessesary audit work/noise for the sake of one extra API call. It's expected that users will want to customise their enabled audit events anyway.